packages/mobile/src/invite/saga.ts

```diff
@@ -192,6 +197,30 @@ export function* sendInvite(
       Logger.error(TAG, 'Currently only dollar escrow payments are allowed')
     }
 
+    // OPEN QUESTION - does storing invitee data only in the event of a successful transaction have
```

Comment:
I think there is potentially an issue where the transaction might be broadcast to the network and added to a block but you lose network connectivity or something so the error handling fails in web3

Reply:
That's true. Doesn't storing the invitee data before we know the transaction was successful open us up to some unintended consequences (e.g., prompting the user to remind their invitee to redeem their escrowed payment with info that isn't valid)? As I see it, downside of failing to store invitee data (e.g., no reminders) is probably more palatable? Also from a frequency perspective, I'd assume it's more likely that one of the two blockchain transactions we are making fails than a connectivity issue arises in between a successful transaction and writing to the store.

If we agree that we don't need to store tx data in the InviteeDetails, I actually think the ideal location to store invite data is in between the invite transaction and and the escrow transaction.

Reply:
I think we can still handle this by storing before we try to send, and then if we are confident the transaction failed we can remove the stored data. I think we want to avoid the case where you send some money and it's not recoverable, but if we at least store the data that means we could get it back at some point. You could guard against this by checking that the account actually has money and it hasn't been redeemed before showing the reminder.

Reply:
That's a good point I hadn't considered. I'll move `storeInvite` function back to where it was, before any transactions are made. I'll also open up issue to deal with removal in the event of a transaction failure.